algoliasearch-core/src/main/java/com/algolia/search/models/rules/Consequence.java

```diff
@@ -60,9 +60,19 @@ public Consequence setFilterPromotes(Boolean filterPromotes) {
     return this;
   }
 
+  public RenderingContent getRenderingContent() {
+    return renderingContent;
+  }
+
+  public Consequence setRenderingContent(RenderingContent renderingContent) {
+    this.renderingContent = renderingContent;
+    return this;
+  }
+
   private ConsequenceParams params;
   private List<ConsequencePromote> promote;
   private Map<String, Object> userData;
   private List<Hide> hide;
   private Boolean filterPromotes;
+  private RenderingContent renderingContent;
```

Comment:
renderingContent is actually _inside_ params in the consequence

Reply:
Nice catch! fixed :)